pkg/splunk/client/awss3client.go

```diff
@@ -195,6 +205,34 @@ func (awsclient *AWSS3Client) GetAppsList() (S3Response, error) {
 	return s3Resp, nil
 }
 
+// DownloadApp downloads the app from remote storage to local file system
+func (awsclient *AWSS3Client) DownloadApp(remoteFile, localFile string) (bool, error) {
+	scopedLog := log.WithName("DownloadApp")
+
+	var numBytes int64
+	file, err := os.Create(localFile)
+	if err != nil {
+		scopedLog.Error(err, "Unable to open file", "localFile", localFile)
```

Comment:
nit: should this be lower case too?

Reply:
May be a re-run of the static analysis is needed to detect these

Reply:
I think it only checks for error messages in `fmt` and not the custom error strings.